src/client/app/components/about.ts

```diff
@@ -13,15 +16,20 @@ import {Strings, getAvailableLanguages, getDisplayLanguage, setDisplayLanguage }
                     <div class="about__primary-text ms-font-xxl">{{config?.build?.name}}</div>
                     <div class="profile__tertiary-text ms-font-m">{{strings.userId}}: ${storage.user}</div>
                     <div class="about__secondary-text ms-font-l">Version: {{config?.build?.version}}
-                    <br/><span class="ms-font-m">(Deployed {{config?.build?.humanReadableTimestamp}})</span>
-                    <br/><span class="ms-font-m">{{config?.editorUrl}}</span>
+                        <br/><span class="ms-font-m">(Deployed {{config?.build?.humanReadableTimestamp}})</span>
                     </div>
                     <pre class="about__tertiary-text ms-font-m">{{cache}}</pre>
                     <div class="about__language">
                         <select class="about__language-select ms-font-m" [(ngModel)]="currentChosenLanguage" (change)="setLanguage($event.target.value)">
                             <option *ngFor="let l of availableLanguages" [value]="l.value">{{l.name}}</option>
                         </select>
                     </div>
+                    <div class="about__environment">
+                        <label class="ms-font-m about__environment-text">{{strings.aboutCurrentEnvironment}}</label>
+                        <select class="about__environment-select ms-font-m" [(ngModel)]="selectedConfig" (ngModelChange)="changeConfig($event)">
+                            <option *ngFor="let c of configs" [value]="c.value">{{c.name}}</option>
```

Comment:
I am not seeing it...
![image](https://user-images.githubusercontent.com/3244360/27978333-ba420e08-6323-11e7-8f76-4b50c8a432a7.png)